onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -176,10 +176,30 @@ def aten_align_to(self: TensorType, names: Sequence[str]) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_all(self: TensorType) -> TensorType:
+@torch_op("aten::all")
```

Comment:
Is this the same with any where we can create a few overloads? Since there isn’t a good way of determining whether an input is a static optional yet in I function I feel it’s better for us to avoid using optionalhaselement on static optional inputs now and should rather create overloads like we have in eg min